README.md

```diff
@@ -285,6 +285,24 @@ EventBridge Rule dashboard widgets show:
 |--|--|
 |![FailedInvocations](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/eventBridgeFailedInvocations.png)|![Invocations](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/eventBridgeInvocations.png)|
 
+### Application Load Balancer
+Application Load Balancer alarms are created for:
+1. HTTP Code ELB 5XX Count
+2. Rejected Connection Count
+3. HTTP Code Target 5XX Count
+4. UnHealthy Host Count
+5. Lambda Internal Error
+6. Lambda User Error
+
+Application Load Balancer dashboard widgets show:
+
+|HTTP Code  ELB 5XX Count|Rejected Connection Count|HTTP Code Target 5XX Count| 
+|--|--|--|
+|![HTTPCode_ELB_5XX_Count](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/httpCodeElb5XXCount.png)) |![RejectedConnectionCount]() |![HTTPCode_Target_5XX_Count](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/httpCodeTarget5XXCount.png)|
+|**UnHealthy Host Count**|**Lambda Internal Error**|**Lambda User Error**|
+|--|--|--|
+|![UnHealthyHostCount](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/unHealthyHostCount.png) |![LambdaInternalError]() |![LambdaUserError](https://raw.githubusercontent.com/fourtheorem/slic-watch/main/docs/lambdaUserError.png)|
```

Comment:
If we cannot provide integration tests for **LambdaInternalError** and **RejectedConnectionCount** we should take out these 2 metrics from the screenshoots section.